contracts/src/examples/cross-token/USDCMock.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {ERC20} from "openzeppelin-contracts/token/ERC20/ERC20.sol";
+import {Ownable} from "openzeppelin-contracts/access/Ownable.sol";
+
+contract USDCMock is ERC20, Ownable {
```

Comment:
Thanks - we have since updated the design to have a USDCTest token designed to simulate USDC and the subnet token will be encapsulated in **IpcTokenReplica**